_posts/2021-02-01-acidanthera-february.md

```diff
@@ -0,0 +1,108 @@
+ 
+The only more important thing to security is what the end-user interacts with, at least for some :P. This is why [@mhaeuser](https://github.com/mhaeuser) continued his OpenCanopy project introducing item scrolling with a large amount of boot options, background image support, which can also be used as a logo through transparency and background colour, and relaxed image dimensions, which of course are well-documented in the PDF spec following our usual standards. Marginal usability improvements also cover [@PMheart](https://github.com/PMheart) additions to ocvalidate, which many found really helpful, and [@GoldFish](https://github.com/GoldFish) audio configuration additions to SysReport. Besides that, we got two new drivers: OpenHfsPlus and OpenPartitionDxe. OpenHfsPlus is a tireless effort by [@shchuko](https://github.com/shchuko), who continued the [@gsomlo](https://github.com/gsomlo) and [@nms42](https://github.com/nms42) work, by finishing a poison-free non-GPL HFS+ driver. While it still is inferior to Apple driver, its speed is clearly better than VBoxHfs, which is now deprecated. OpenPartitionDxe on the other side improves compatibility with legacy macOS recoveries.
+ 
+Sometimes the changes are not dictated by the necessity to improve things on our end or even our bugs, but rather flaws in the design & code of the other parties. This release includes three changes that happened due to exactly this kind of issues. One of them is the result of Apple’s APFS design, which does not track trimmed areas, resulting in essentially broken trim support for a vast amount of third-party SSDs. The discovery and an ugly workaround are described in the [SetApfsTrimTimeout section](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Configuration.pdf) thanks to [@lvs1974's](https://github.com/lvs1974) efforts. The second issue is a design flaw of the older SMBIOS specifications, which caused confusion of how UUIDs can be read and written. While no changes are needed for the existing users, some new users might benefit from the improvements in dmidecode and the new [SetApfsTrimTimeout option](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Configuration.pdf). Last but not least was a discovery that some firmwares disable the support for UEFI driver loading, even with UEFI Secure Boot disabled. This obviously violates the UEFI specification, but [Microsoft](https://github.com/badstorm/surface-pro-7-opencore/issues/9) thinks differently, breaking the entire loading of OpenCore. If any Microsoft Surface firmware developers are reading this, it will help if you provide a comment on the matter, since even the provided workaround by transitioning to the app only uncovered further issues.
```

Comment:
```suggestion
Sometimes the changes are not dictated by the necessity to improve things on our end or even our bugs, but rather flaws in the design & code of the other parties. This release includes three changes that happened due to exactly this kind of issues. One of them is the result of Apple’s APFS design, which does not track trimmed areas, resulting in essentially broken trim support for a vast amount of third-party SSDs. The discovery and an ugly workaround are described in the [SetApfsTrimTimeout section](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Configuration.pdf) thanks to [@lvs1974's](https://github.com/lvs1974) efforts. The second issue is a design flaw of the older SMBIOS specifications, which caused confusion of how UUIDs can be read and written. While no changes are needed for the existing users, some new users might benefit from the improvements in dmidecode and the new [SetApfsTrimTimeout option](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Configuration.pdf). Last but not least was a discovery that some firmwares disable support for UEFI driver loading, even with UEFI Secure Boot disabled. This obviously violates the UEFI specification, but [Microsoft](https://github.com/badstorm/surface-pro-7-opencore/issues/9) thinks differently, breaking the entire loading of OpenCore. If any Microsoft Surface firmware developers are reading this, it will help if you provide a comment on the matter, since even the provided workaround by transitioning to the app only uncovered further issues.
```